src/node_report.cc

```diff
@@ -379,7 +380,7 @@ void SetCommandLine() {
  *   const char* location
  *   char* name - in/out - returns the report filename
  ******************************************************************************/
-void TriggerNodeReport(Isolate* isolate, DumpEvent event, const char* message, const char* location, char* name) {
+void TriggerNodeReport(Isolate* isolate, DumpEvent event, const char* message, const char* location, char* name, v8::MaybeLocal<v8::Value> error) {
```

Comment:
The comment before this line documents the parameters. `error` should probably be added there.